dotCMS/src/main/java/com/dotmarketing/util/UtilMethods.java

```diff
@@ -1273,6 +1273,7 @@ public static String espaceForVelocity(String text) {
         if (isSet(text)) {
             text = replace(text, "\"", "${quote}");
```

Comment:
If this method is highly used, you may consider to use a StringBuilder rather than just a spring, a new string is created in any replacement.